manifests/05-operator.yaml

```diff
@@ -41,6 +41,10 @@ spec:
           value: docker.io/openshift/origin-console:latest
         - name: OPERATOR_NAME
           value: "console-operator"
+        - name: POD_NAME
```

Comment:
Do you happen to remember why this was needed?  We talked about it on Bluejeans, I just forget atm.

Reply:
I believe you mentioned it does affect CI.

Reply:
It's because of the Recorder. The envar itself is explained [here](https://github.com/openshift/library-go/blob/af91f0cfd1aad763ce48455b4d34e927dc782f04/pkg/operator/events/recorder.go#L24-L26)

Because of that we needed to add the `pods` && `replicaset` to the rbac rule, so in order to be able to record events, the console-operator pod (or better to say it's service account) needs to be able to get that specified pod and also replicaset.

Reply:
That would mean you need read only access to those resources, not read/write.

Reply:
Great, sounds good.